Plugin/MyHomes/src/main/java/org/spongepowered/cookbook/myhomes/MyHomes.java

```diff
@@ -0,0 +1,53 @@
+package org.spongepowered.cookbook.myhomes;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.cookbook.myhomes.data.Keys;
+import org.spongepowered.cookbook.myhomes.data.friends.impl.FriendsDataBuilder;
+import org.spongepowered.cookbook.myhomes.data.friends.impl.FriendsDataImpl;
+import org.spongepowered.cookbook.myhomes.data.friends.impl.ImmutableFriendsDataImpl;
+import org.spongepowered.cookbook.myhomes.data.home.Home;
+import org.spongepowered.cookbook.myhomes.data.home.HomeData;
+import org.spongepowered.cookbook.myhomes.data.home.ImmutableHomeData;
+import org.spongepowered.cookbook.myhomes.data.home.impl.HomeBuilder;
+import org.spongepowered.cookbook.myhomes.data.home.impl.HomeDataBuilder;
+import org.spongepowered.api.data.DataManager;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.game.state.GameInitializationEvent;
+import org.spongepowered.api.event.network.ClientConnectionEvent;
+import org.spongepowered.api.plugin.Plugin;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.chat.ChatTypes;
+import org.spongepowered.api.text.format.TextStyles;
+
+@Plugin(id = "myhomes")
+public class MyHomes {
+
+    @Listener
+    public void onGameInitialization(GameInitializationEvent event) {
+        DataManager dm = Sponge.getDataManager();
+
+        // FriendsData
+        dm.register(FriendsDataImpl.class, ImmutableFriendsDataImpl.class, new FriendsDataBuilder());
+
+        // Home
+        dm.registerBuilder(Home.class, new HomeBuilder());
```

Comment:
Should note that `Home` as a `DataSerializable` should be registered before any data manipulator that relies on it can be registered. The reasoning is that in SpongePowered/SpongeAPI#1198, the system will need to know of all of the serializable objects and translatable objects before it can generate the appropriate serialization strategies for custom data.